workspace/__lib__/xod/core/clock/patch.test.tsv

```diff
@@ -11,3 +11,11 @@ __time(ms)	IVAL	EN	RST	TICK
 500	1	true	pulse	no-pulse
 1001	1	true	no-pulse	no-pulse
 1501	1	true	no-pulse	pulse
+0	0	true	no-pulse	pulse
```

Comment:
A comment above the block telling what we’re testing (edge-case intervals) would be helpful